### PR TITLE
Latest UI changes to SideMenu and FarmPage

### PR DESCRIPTION
**Pull request by RudeWolfCoding**
**PLEASE DO NOT SUBMIT TOKEN ADDITIONS AS PULL REQUESTS**

All token requests should be made via an issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -101,13 +101,13 @@
     "webfontloader": "^1.6.28"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "start:prod": "env-cmd -f .env.production react-scripts start",
+    "start": "SET NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
```
Comment (leonprou) [inline package.json:104]: why that is needed?

File: src/pages/Farms/index.tsx [hunk 2]
```diff
@@ -43,14 +75,20 @@ export default function Farms() {
         <Wrapper>
           <div>
             <Header>Farm</Header>
+            <SubHeader>Let&apos;s farm FUSE and VOLT with your LP tokens!</SubHeader>
             <SubHeader>
-              Let&apos;s farm FUSE with your LP tokens!{' '}
-              <ExternalLink href="https://oldrewards.fuse.io">(Click here for the old faming page)</ExternalLink>
+              <Arrow />{' '}
```
Comment (leonprou) [inline src/pages/Farms/index.tsx:80]: the link should be of type docs.fuse.io

**Review by leonprou — CHANGES_REQUESTED** [review]
Good work Rudy 👍 
Can you go over the comment?

File: package.json (hunk 1 above)
Comment (RudeWolfCoding) [reply package.json:104]: Latest NPM version will fail on windows without it, can be removed.

File: src/pages/Farms/index.tsx (hunk 2 above)
Comment (RudeWolfCoding) [reply src/pages/Farms/index.tsx:80]: This is no longer relevant as it has been completely removed and the links has been updated.

**Review by leonprou — APPROVED** [review]

**Pull request merged**